eng/scripts/Set-VcpkgWriteModeCache.ps1

```diff
@@ -0,0 +1,17 @@
+param(
+    [string] $StorageAccountKey
+)
+Install-Module "Az.Storage" -AllowClobber -Force
```

Comment:
@weshaggard -- The ps module helper fails when attempting to install `Az.Storage` on windows agents. I think we should add some kind of `-AllowClobber` feature

Example failure: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1454990&view=logs&j=34680a5f-6ae3-5138-ac88-c2c786351f1e&t=2a1986a3-0bcc-56b5-60f7-9b7c19a7a9f7&l=72

```
Installing module Az.Storage with min version 4.3.0 from https://www.powershellgallery.com/api/v2
PackageManagement\Install-Package : The following commands are already available on this system:'Start-CopyAzureStorageBlob,Stop-CopyAzureStorageBlob'. This module 'Az.Storage' may override the existing commands. If you still want to install this module 'Az.Storage', use -AllowClobber parameter.
At C:\program files\powershell\7\Modules\PowerShellGet\PSModule.psm1:9711 char:34
+ … talledPackages = PackageManagement\Install-Package @PSBoundParameters
+                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (Microsoft.PowerShel…lets.InstallPackage:InstallPackage) [Install-Package], Exception
+ FullyQualifiedErrorId : CommandAlreadyAvailable,Validate-ModuleCommandAlreadyAvailable,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage
##[debug]Exit code: 1
##[debug]Leaving Invoke-VstsTool.
##[error]PowerShell exited with code '1'.
```